### DISCUSSION TITLE
When the data processing logic is relatively complex, if the backend processing takes more than 40 seconds, an error will be reported

### DISCUSSION THREAD

**Discussion by oz-hjx210** [filamentphp/filament#6272] [Help]
![image](https://user-images.githubusercontent.com/120376499/233524327-49d2c3cb-1e76-4901-bdc6-87a67311d6ab.png)

**Comment by danharrin** [comment]
Requests should **never** take 40 seconds. Please use a Laravel queued job.

**Comment by oz-hjx210** [comment]
how can i use Laravel queued job. in filament ,can you show me a demo code

or if i want to   modify   40 seconds  to  60 seconds.   where can i config it  .  thanks

**Reply by danharrin** [reply]
You can use a queued job no differently than how you would in Laravel. I do not have time to write code for you, lots of information about jobs is available in tutorials and the Laravel documentation.
